shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ddl/drop-database-link.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<sql-cases>
+    <sql-case id="drop_database_link" value="DROP PUBLIC DATABASE LINK remote USING 'remote'" db-types="Oracle" />
+    <sql-case id="drop_database_link_connect_to_user" value="DROP DATABASE LINK local CONNECT TO hr IDENTIFIED BY password USING 'local';" db-types="Oracle" />
+    <sql-case id="drop_database_link_connect_to_current_user" value="DROP DATABASE LINK remote.us.example.com CONNECT TO CURRENT_USER USING 'remote';" db-types="Oracle" />
```

Comment:
Please correct these test cases according to the `drop database link` grammar

Reply:
Made the changes